frontend/jest.config.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
   testMatch: ['**/?(*.)+(spec).js'],
   collectCoverage: true,
   collectCoverageFrom: ['src/**/*', '!src/assets/**'],
-  transformIgnorePatterns: ['/node_modules/(?!(vue-scrollactive)/)']
+  transformIgnorePatterns: ['/node_modules/(?!(vue-scrollactive)/)'],
```

Comment:
I don't know. But because vue-scrollactive supported was dropped, I ended up rewriting scroll logic to vanilla js. In the newest PR I removed vue-scrollactive, to avoid confusion. Thank you for pointing that out.